Add Arabic, Arabian Peninsula Sky Cultures and replace Arabic Moon stations by Arabic Lunar Stations Sky Culture

kajaji opened the pull request:
### Description
Two new sky cultures have been added: 
1. **Arabic**, which is the indigenous Ancient Arabs sky culture extracted from books written more than 10 centuries ago.
2.  **Arabian Peninsula**, the folk sky culture of the region covering central Arabian Peninsula and other nearby regions. The source of this sky culture are collections of folk poetry and proverbs and verbal communication of oral traditions.
 
**Arabic Lunar Stations** is a revised replacement of the old Arabic Moon Stations. This sky culture is still popular and in active use until present in some Arab countries.

I have updated the following files to reflect the new additions:
- skycultures/CMakeLists.txt
- po/stellarium-skycultures/POTFILES.in
- util/skycultures/po4a.config

### Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] This change requires a documentation update

### How Has This Been Tested?
Tested on Stellarium 0.21.2 running on windows computer.
**Test Configuration**:
* Operating system: <Windows 10>
* Graphics Card: <NVidia, Geforce GTX 1050

## Checklist:
- [ ] My code follows the [code style](http://stellarium.org/doc/head/codingStyle.html) of this project.
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

alex-w commented on util/skycultures/po4a.config:
```diff
@@ -1,6 +1,9 @@
+[type: xhtml] ../../skycultures/arabian_peninsula/description.en.utf8
+[type: xhtml] ../../skycultures/arabic/description.en.utf8
+[type: xhtml] ../../skycultures/arabic_lunar_stations/description.en.utf8
 [type: xhtml] ../../skycultures/arabic_moon_stations/description.en.utf8
```
The sky culture arabic_moon_stations is removed and it should be removed from po4a.config file also.

kajaji replied:
Thanks Alex,
Done.

alex-w reviewed (changes requested):
The file `skycultures/arabic_lunar_stations/constellation_names.eng.fab` has wrong format - please fix it.

alex-w commented:
Please update file `src/translations.h`

kajaji commented:
@alex-w 
> Please update file `src/translations.h`

I couldn't find any relevance in the file.

alex-w commented on skycultures/arabic_lunar_stations/dso_names.fab:
```diff
@@ -0,0 +1,13 @@
+NGC 869 |_("The Wrist 1")
+NGC 884 |_("The Wrist 2")
+NGC 869 |_("The She-Camel's Brand 1")
+NGC 884 |_("The She-Camel's Brand brand 2")
```
Is it correct?

kajaji replied:
Yes.
I started including references in the constellation names, but I got  [issue 2055](https://github.com/Stellarium/stellarium/issues/2055).
I will add the references since this issue is resolved and test with  0.21.2-07bea75.

gzotti replied:
I think the question was about "Brand brand" (repeated word)

kajaji replied:
Thanks @gzotti 
Fixed.

alex-w reviewed (commented):
Do you have `references.fab` files in sky cultures and this is good news, but all those references related to whole SC probably, because I see no links in `star_names.fab` files. Is it correct ?

alex-w reviewed (commented):
Please reduce the size of image

alex-w commented on skycultures/arabian_peninsula/description.en.utf8:
```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+<head>
+<title>Arabian Peninsula</title>
+</head>
```
By formal rules no need header block for HTML'ed description, because we includes the descripting in our HTML page.

alex-w commented on skycultures/arabian_peninsula/description.en.utf8:
```diff
@@ -0,0 +1,115 @@
+<p id="footnote-2">[2]: Le Bon, Gustave, La civilisation des Arabes, Paris, Librairie de Firmin-Didot, 1884, livre premier, pg. 1.
+<p id="footnote-3">[3]: AlAjaji, Khalid, The Explanation of Al-Khalawi Astronomy Poems, Riyadh,  Saudi arabia, 2018.
+<p id="footnote-4">[4]: AlAjaji, Khalid, The Explanation of Al-Qadhi Poem Anwa'a and Stars, Riyadh,  Saudi arabia, 2013.</p>
+</html>
```
No need `</html>` here

alex-w commented on skycultures/arabic/description.ar.utf8:
```diff
@@ -0,0 +1,283 @@
+<!DOCTYPE html>
+<html lang="ar" dir="rtl">
+<head>
+<title>العربي</title>
+</head>
```
By formal rules no need header block for HTML'ed description, because we includes the descripting in our HTML page.

alex-w commented on skycultures/arabic/description.ar.utf8:
```diff
@@ -0,0 +1,283 @@
+<h2>الحاشية</h2>
+<p id="footnote-1">[1]: أبو محمد عبد الله بن مسلم ابن قتيبة الدينوري (توفي 276 هـ)، كتاب الأنواء في مواسم العرب، دائرة المعارف العثمانية، حيدر أباد، الهند، 1375 هـ.
+<p id="footnote-2">[2]: كتاب الكواكب لأبي الحسين عبد الرحمن بن عمر الرازي المعروف بالصوفي، تحقيق خالد بن عبد الله العجاجي، نسخة رقمية، 1442 هـ.</p>
+</html>
```
No need `</html>` here

alex-w commented on skycultures/arabic/description.en.utf8:
```diff
@@ -0,0 +1,296 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+<head>
+<title>Arabic</title>
+</head>
```
By formal rules no need header block for HTML'ed description, because we includes the descripting in our HTML page.

alex-w commented on skycultures/arabic/description.en.utf8:
```diff
@@ -0,0 +1,296 @@
+<p>Attribution-NonCommercial-NoDerivatives 4.0 International (CC BY-NC-ND 4.0). The author wishes that this work be cited according to the common rules.</p>
+<p id="footnote-1">[1]: Ibn Qutaybah (Arabic ابن قتيبة) (b. 828 d. 884 or 889), Kitab Al-Anwā’a (Arabic الأنواء في مواسم العرب), (Arabic print of the original book in 1956 in India Hydarabad al-Dakan), page: يد.</p>
+<p id="footnote-2">[2]: <a href="https://drive.google.com/drive/folders/1s6JXzftwjMQ5rgZoGE3718EtBLBZtjzr?usp=sharing">Book of the Stars: Critical edition with commentary by Khalid AlAjaji, digital edition</a></p>
+</html>
```
No need `</html>` here

alex-w commented on skycultures/arabic/dso_names.fab:
```diff
@@ -0,0 +1,13 @@
+NGC 884 |_("The She-Camel's Brand 2")
+# TRANSLATORS: Al-Thurayya is a proper name
+M 45    |_("Al-Thurayya")
+M 44    |_("The Lion'S Nostrils")
```
A typo? Probably it should be `The Lion's Nostrils`

alex-w commented on skycultures/arabic_lunar_stations/description.ar.utf8:
```diff
@@ -0,0 +1,193 @@
+<!DOCTYPE html>
+<html lang="ar" dir="rtl">
+<head>
+<title>منازل القمر العربية</title>
+</head>
```
By formal rules no need header block for HTML'ed description, because we includes the descripting in our HTML page.

alex-w commented on skycultures/arabic_lunar_stations/description.ar.utf8:
```diff
@@ -0,0 +1,193 @@
+<p id="footnote-10">[10]: كتاب الكواكب لأبي الحسين عبد الرحمن بن عمر الرازي المعروف بالصوفي، تحقيق خالد بن عبد الله العجاجي.</p>
+<p id="footnote-11">[11]: Schjellerup, Description des etoiles fixes, 1874, p. 163. (Translation into French)</p>
+<p id="footnote-12">[12]: Laffitte, Roland, Le Ciel des Arabes, Geuthner, Paris 2012
+</html>
```
No need `</html>` here

alex-w commented on skycultures/arabic_lunar_stations/description.en.utf8:
```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+<head>
+<title>Arabic Lunar Stations</title>
+</head>
```
By formal rules no need header block for HTML'ed description, because we includes the descripting in our HTML page.

alex-w commented on skycultures/arabic_lunar_stations/description.en.utf8:
```diff
@@ -0,0 +1,151 @@
+<p id="footnote-10">[10]: Book of the Stars by Al-Sufi (died 986): Critical edition with commentary by Khalid AlAjaji, digital edition, 2021 AD, كتاب الكواكب لأبي الحسين عبد الرحمن بن عمر الرازي المعروف بالصوفي (291-376)، تحقيق خالد بن عبد الله .</p>
+<p id="footnote-11">[11]: Schjellerup, Description des etoiles fixes, 1874, p. 163. (Translation into French)</p>
+<p id="footnote-12">[12]: Laffitte, Roland, Le Ciel des Arabes, Geuthner, Paris 2012
+</html>
```
No need `</html>` here

alex-w commented:
> Please reduce the size of image

I meant reduce physical size of file `arabian_peninsula.png` and probably JPG will be better here.

kajaji commented:
@alex-w 
fixed html headers and typo.

alex-w commented on skycultures/arabic/constellation_names.eng.fab:
```diff
@@ -0,0 +1,138 @@
+1903 "يدا الناقة" _("Legs Of Tthe She-Came")
+#The name is left blank intentionally, label of 2604 Al-Thurayya is sufficient. This constellation is to show the complete Al-Thurayya image with its two hands.
+#2000 "الثريا " _("Al-Thurayya")
+2000 " " _("")
```
Why it empty?

kajaji replied:
This constellation has a small part (2604) with the same name and the label is better placed on the screen.
https://github.com/kajaji/stellarium/blob/c0882bae2304b2662464ebc6335eeccb82222717/skycultures/arabic/constellation_names.eng.fab#L65
The bigger part (2000) label would go inside another constellation and could cause confusion.
![Thurayya](https://user-images.githubusercontent.com/34771982/143844925-b7a045e4-9929-440b-bf47-89482cd62f77.jpg)

alex-w commented on skycultures/arabic/constellation_names.eng.fab:
```diff
@@ -0,0 +1,138 @@
+2604 "الثريا" _("Al-Thurayya")
+#This constellation defines the boundary of Al-Dabaran, his camels and two dogs. Name is intentionally left blank
+#2700 "الدبران وقلاصه وكلباه" _("Al-Dabaran, His Camels And Two Dogs")
+#2700 " " _(" ")
```
Why it empty?

kajaji replied:
This one is commented out.
It is kept for reference

alex-w commented on skycultures/arabic/star_names.fab:
```diff
@@ -0,0 +1,179 @@
+ 11020|_("The Wrist")
+# TRANSLATORS: Al-Ayyuq is a proper name meaning the obstructor
+ 24608|_("Al-Ayyuq")
+# TRANSLATORS: Ayyuq Al-Thurayya is a proper name meaning the obstructorof Al-Thurayya
```
Please fix the typo

kajaji replied:
fixed

alex-w commented on skycultures/arabic/star_names.fab:
```diff
@@ -0,0 +1,179 @@
+ 87073|_("Tail Segments Of The Scorpion 6")
+ 86670|_("Tail Segments Of The Scorpion 7")
+ 85927|_("Raised Tail Of The Scorpion")
+# 85696|_(" ")
```
Why it empty?

kajaji replied:
It is a comment.
A common name is given to HIP 85927 and HIP 85696. The two stars are very close to each other, so one label is used to mimimize clutter and I kept the other as a comment for documentation.

gzotti commented on skycultures/arabic/constellation_names.eng.fab:
```diff
@@ -0,0 +1,138 @@
+1900 "الناقة" _("The She-Camel")
+1901 "رأس الناقة" _("Head Of Tthe She-Came")
+1902 "عنق الناقة" _("Neck Of Tthe She-Came")
+1903 "يدا الناقة" _("Legs Of Tthe She-Came")
```
Typo --> ... of the She-Camel

kajaji replied:
Fixed

gzotti commented on skycultures/arabic/constellation_names.eng.fab:
```diff
@@ -0,0 +1,138 @@
+1400 "قصعة المساكين" _("The Bowl Of The Poor")
+1500 "الروضة والأغنام" _("The Desert Garden And The Goats")
+1501 "النسق الشآمي" _("The Northern Line")
+1502 "النسق اليماني" _("اthe Southern Line")
```
strange first character in translatable string. Maybe "The ..."

kajaji replied:
Sorry, remnant of the original native language file.
Fixed.
I added references. Beta75 solves issue #2055 .

gzotti commented on skycultures/arabic/constellation_names.eng.fab:
```diff
@@ -0,0 +1,138 @@
+2502 "الفرغ المقدم" _("The Front Bucket Mouth")
+2504 "الفرغ المؤخر" _("The Rear Bucket Mouth")
+2506 "الرشاء" _("The Bucket Rope")
+#For reference 2 see vol. 1 pg.
```
where is ref2 labelled, which page?

kajaji replied:
uploaded constellation names with references.

gzotti commented on skycultures/arabic/constellation_names.eng.fab:
```diff
@@ -0,0 +1,138 @@
+4201 "الهقعة" _("The Circular Mark‎")
+4202 "النظم" _("The String")
+4203 "الجواري" _("The Young Maids")
+#Ref 3, Pg.
```
which page?

kajaji replied:
done

gzotti commented on skycultures/arabic/constellation_names.eng.fab:
```diff
@@ -0,0 +1,138 @@
+#2700 "الدبران وقلاصه وكلباه" _("Al-Dabaran, His Camels And Two Dogs")
+#2700 " " _(" ")
+2701 "الدبران" _("Al-Dabaran")
+#Ref 3 pg.
```
which page?

kajaji replied:
done

gzotti commented on skycultures/arabic/constellationship.fab:
```diff
@@ -0,0 +1,242 @@
+﻿#0100 الفرقدان
```
Please use the English names in the comments.

kajaji replied:
Ok, done.

gzotti commented on skycultures/arabic/dso_names.fab:
```diff
@@ -0,0 +1,13 @@
+#NGC 869 |_("المعصم 1")
```
Please use English names. If useful, add an English transliteration. 
Ah, those are commented away. Maybe delete (@alex-w ) ?

kajaji replied:
This is to help me translate back to Arabic.
I added English names.

gzotti commented on skycultures/arabic/dso_names.fab:
```diff
@@ -0,0 +1,13 @@
+# TRANSLATORS: Al-Thurayya is a proper name
+M 45    |_("Al-Thurayya")
+M 44    |_("The Lion's Nostrils")
+M 7     |_("The Scorpion'S Stinger")
```
--> The Scorpion's Sting ?

kajaji replied:
Fixed capital S.
As I checked, sting is the injury, and stinger is tip of the scorpion's tail.

gzotti replied:
"Sting" can be both and is I think more common. But you can leave it.

kajaji replied:
Changed to "Sting">

gzotti commented on skycultures/arabic/reference.fab:
```diff
@@ -0,0 +1,7 @@
+2|Al-Marzoqi, Abū Ali Aḥmad b. Muḥammad b. Al-Ḥassan (died 1030 AD), Al-Azminah wal’Amkinah (Times and Places), (Arabic print of the original book in 2002, World of Books, Beirut, Lebanon)., الإمام أبو علي أحمد بن محمد بن الحسن المرزوقي (توفي سنة 421 هـ)، الأزمنة والأمكنة، تحقيق د. محمد نايف الدليمي، عالم الكتب، بيروت، لبنان، 1422 هـ.|
+3|Al- Thaqafī, Abdu’Allah b. Ḥussayn b. Aṣim (died ~ 1012 AD), Al-Anwā‘ wal’Azminah (Meteorology and times), (Arabic print of the original book in 1996, Dar Al-Jeel, Beirut, Lebanon)., عبد الله بن حسين بن عاصم الثقفي (توفي 403 هـ)، الأنواء والأزمنة، تحقيق: د. نوري حمودي القيسي، د. محمد نايف الدليمي، دار الجيل، بيروت، لبنان، 1416 هـ.|
+4|Book of the Stars by Al-Sufi (died 986): Critical edition with commentary by Khalid AlAjaji, digital edition, 2021 AD, كتاب الكواكب لأبي الحسين عبد الرحمن بن عمر الرازي المعروف بالصوفي (291-376)، تحقيق خالد بن عبد الله العجاجي|https://drive.google.com/drive/folders/1s6JXzftwjMQ5rgZoGE3718EtBLBZtjzr?usp=sharing
+5|Al-farāhīdī, Abu Abdul’Raḥmān Al-ḫalīl b. Aḥmad (died ~ 786 AD), Kitab Al-Ayn (The book of Arabic letter العين), (Arabic print of the original book, 1980 – 1985)., أبو عبد الرحمن الخليل بن أحمد الفراهيدي (توفي 170 ه)، كتاب العين، 1400 – 1405 هـ.|
```
Would you not capitalize as "Al-Farahidi"? (sorry, missing long a/i here)

kajaji replied:
done

gzotti commented on skycultures/arabic/star_names.fab:
```diff
@@ -0,0 +1,179 @@
+ 35550|_("The Claws 4")
+ 34088|_("The Claws 5")
+ 35350|_("The Claws 6")
+ 46750|_("The Eyelashes")
```
no numbers here?

kajaji replied:
done

gzotti commented on skycultures/arabian_peninsula/star_names.fab:
```diff
@@ -0,0 +1,42 @@
+34088|_("The Claws 5")
+35350|_("The Claws 6")
+57632|_("Star Of Weather Change")
+65474|_("The High Un Armed One")
```
Unarmed

kajaji replied:
Fixed and added # TRANSLATORS notes.

gzotti commented on skycultures/arabic_lunar_stations/constellation_names.eng.fab:
```diff
@@ -0,0 +1,55 @@
+2602 "الشرطان" _("The Two Signs")
+2603 "البطين" _("The Little Abdomen")
+2604 "الثريا" _("Al-Thurayya")
+2801 "الذراع المقبوضة" _("Retracted Arm")
```
Most other names are with article. Is there a reason to not write "The Retracted Arm" here?

kajaji replied:
No reason. Originally I had "Retracted Arm [Of The Lion]" to clarify the name. The native name is only "The Retracted Arm".

Fixed.

gzotti commented on skycultures/arabic_lunar_stations/reference.fab:
```diff
@@ -0,0 +1,7 @@
+2|Al-Marzoqi, Abū Ali Aḥmad b. Muḥammad b. Al-Ḥassan (died 1030 AD), Al-Azminah wal’Amkinah (Times and Places), (Arabic print of the original book in 2002, World of Books, Beirut, Lebanon)., الإمام أبو علي أحمد بن محمد بن الحسن المرزوقي (توفي سنة 421 هـ)، الأزمنة والأمكنة، تحقيق د. محمد نايف الدليمي، عالم الكتب، بيروت، لبنان، 1422 هـ.|
+3|Al- Thaqafī, Abdu’Allah b. Ḥussayn b. Aṣim (died ~ 1012 AD), Al-Anwā‘ wal’Azminah (Meteorology and times), (Arabic print of the original book in 1996, Dar Al-Jeel, Beirut, Lebanon)., عبد الله بن حسين بن عاصم الثقفي (توفي 403 هـ)، الأنواء والأزمنة، تحقيق: د. نوري حمودي القيسي، د. محمد نايف الدليمي، دار الجيل، بيروت، لبنان، 1416 هـ.|
+4|Book of the Stars by Al-Sufi (died 986): Critical edition with commentary by Khalid AlAjaji, digital edition, 2021 AD, كتاب الكواكب لأبي الحسين عبد الرحمن بن عمر الرازي المعروف بالصوفي (291-376)، تحقيق خالد بن عبد الله العجاجي|https://drive.google.com/drive/folders/1s6JXzftwjMQ5rgZoGE3718EtBLBZtjzr?usp=sharing
+5|Al-farāhīdī, Abu Abdul’Raḥmān Al-ḫalīl b. Aḥmad (died ~ 786 AD), Kitab Al-Ayn (The book of Arabic letter العين), (Arabic print of the original book, 1980 – 1985)., أبو عبد الرحمن الخليل بن أحمد الفراهيدي (توفي 170 ه)، كتاب العين، 1400 – 1405 هـ.|
```
again, capitalize author name. (also al-Halil with small al- and capital H ? I am no expert in these things, though.) If yes, please check the other names for these details.

kajaji replied:
Prof. Paul Kunitzsch uses proper names with small al-.  I found these two links in agreement with Prof. Kunitzsch usage:
- [al- Arabic language](https://www.britannica.com/topic/al-Arabic-language)
- [The Chicago Manual of Style, sixteenth edition (2010), The Arabic definite article](https://english.stackexchange.com/questions/498568/i-am-trying-to-find-out-if-there-is-a-convention-of-correctness-for-writing-arab)
I will try to conform to this in all relevant files.

gzotti replied:
Thank you. Yes, I have a few of Kunitzsch's works, so I got to use that style.

gzotti commented on skycultures/arabic_lunar_stations/star_names.fab:
```diff
@@ -0,0 +1,146 @@
+35550|_("The Claws 4")
+34088|_("The Claws 5")
+35350|_("The Claws 6")
+46750|_("The Eyelashes")
```
again: with numbers??

kajaji replied:
Fixed.

gzotti commented on skycultures/arabian_peninsula/constellation_names.eng.fab:
```diff
@@ -0,0 +1,40 @@
+# TRANSLATORS: Aba Bzay is a proper name
+100 "الحويجزين - الدخيلين - آل أبا بزي" _("The Two Guards - The Two Refugees - The Family Of Aba Bzay")
```
out of curiosity: is the ID a number code? Object list in some book?

kajaji replied:
No.
I built these sky culture along al-Sufi sky culture beginning with northern constellations. The numbers used are just to facilitate later additions if any, in the same or close by regions. Using abbreviations here would be confusing.
I published two books on this subject, but I didn't have tables or a catalog of folk names.

gzotti commented on skycultures/arabian_peninsula/constellationship.fab:
```diff
@@ -0,0 +1,35 @@
+4202 2 25930 26311 26311 26727 
+2650 2 9884 8796 8796 10064 
+4299 10 26207 26487 25336 26487 26487 27989 25336 25930 25930 26311 26311 26727 26727 27366 27366 24436 24436 25930 27989 26727 
+ 
```
Github tells me "no newline at end of file". Please add one.

kajaji replied:
Fixed, last line had a blank charactes.

gzotti commented on skycultures/arabian_peninsula/description.en.utf8:
```diff
@@ -0,0 +1,109 @@
+<br>عبد الرحمن بن زيد السويداء، شعراء الجبل الشعبيون، الرياض، المملكة العربية السعودية، 1434 هـ.</li>
+<li>AlAjaji, Khalid, The Explanation of al-Qadhi Poem Anwa'a and Stars, Riyadh,  Saudi Arabia, 2013.
+<br>خالد بن عبد الله العجاجي، شرح قصيدة القاضي في الأنواء والنجوم، الرياض، المملكة العربية السعودية، 1434 هـ.</li>
+<li>AlAjaji, Khalid, The Explanation of al-Khalawi Astronomy Poems, Riyadh,  Saudi arabia, 2018.
```
capitalize "Arabia".

kajaji replied:
done

gzotti commented on skycultures/arabian_peninsula/description.en.utf8:
```diff
@@ -0,0 +1,109 @@
+<h5 id="mosque-of-al-thurayya">Mosque of al-Thurayya</h5>
+<p>A triangle of stars that rise before al-Thurayya (The Pleiades). It is composed of the three stars: α Ari, α Tri, and β Tri. The source of this name is oral tradition in the region of Wadi al-Dawasir in southwestern Saudi Arabia.</p>
+<h5 id="northern-claw-of-al-jawza-southern-claw-of-al-jawza-claws-of-al-jawza">Northern Claw of al-Jawza, Southern Claw of al-Jawza, Claws of al-Jawza</h5>
+<p>Names of the stars: α, γ, β, and κ Ori, orally communicated from the Qasseem region of Saudi Arabia. The specific name <em>Northern Claw of al-Jawza</em> appeared in the book<em>Hisorty of Najd</em> by the historian Ibn Bishr (died 1873 AD). This might mean the two stars: α, and γ Ori, while Southern claw of al-Jawza would be the two stars: β, and κ Ori</p>
```
Booktitle "History"?

kajaji replied:
done

gzotti commented on skycultures/arabian_peninsula/description.en.utf8:
```diff
@@ -0,0 +1,109 @@
+</ol>
+<h2 id="constellations">Constellations</h2>
+<h5 id="al-nayem-the-seven-daughters-of-na-sh">Al-Nayem - The Seven - Daughters of Na'sh</h5>
+<p>(Arabic النعايم), meaning is not known for certain, composed of the seven stars of the Big Dipper. other names are: <em>The Seven</em> (Arabic السبَّع) and <em>Daughters of Na'sh</em> (Arabic بنات نعش). The first name is common in oral poetry.</p>
```
Can you please add an English transliteration to the Arabic phrases? It helps most interested people reading the original words. I think some spelling like 

`(Arabic النعايم <it>al-nayem</it>)`

would be nice. (I never can read the loop-like letter between n and a!)

kajaji replied:
We can do this for the names in all Arabic sky cultures. The difficulty is that using one of the standards for transliterations needs some learning by the reader. For example:

```(Arabic النعايم _an-Naʿayem_)```

Using ʿ to indicate the letter (عـ) in Arabic. ʿ or sometimes superscript c is similar in shape to the Arabic letter, but far from how that letter is pronounced. This is used by prof. Paul Kunitzsch and Roland Laffitte.

This is the standard I used when working with Laffitte:

| Arabic letter | Transliteration | Arabic letter | Transliteration | Arabic letter | Transliteration |
|---------------|-----------------|---------------|-----------------|---------------|-----------------|
| ء | ‘ | ر | r | ف | f |
| ا | ā | ز | z | ق | q |
| ب | b | س | s | ك | k |
| ت | t | ش | š | ل | l |
| ث | ṯ | ص | ṣ | م | m |
| ج | ğ | ض | ḍ | ن | n |
| ح | ḥ | ط | ṭ | ه | h |
| خ | ḫ | ظ | ẓ | و | w |
| د | d | ع | c | ي | y |
| ذ | ḏ | غ | ġ |   |   |

Short vowels : a, i, u
Long vowels : ā, ī, ū

transliteration of /ج/ pronounced as the j in _Jim_by /ğ/ (instead of /j/): so the letter can be read easily not only in French or English but also in German, Spanish, Italian, etc.

To simplify reading, this variation can be used:
 /ت/ and  /ط/ → /t/
 /ث/ → /th/
 /ح/ and  /ه/ → /h/
 /خ/ → /kh/
/ذ/ and  /ض/ → /dh/
/ش/ → /sh/
 /غ/ → /gh/

For more, see [Romanization of Arabic](https://en.wikipedia.org/wiki/Romanization_of_Arabic).

gzotti replied:
Excellent. I would prefer Laffite/Kunitzsch, as they are more accurate. Translators to other languages may then adopt their localized transliterations (hopefully in concordance with the single-names...)

kajaji replied:
I will start this, but I expect it to take some time because of the absence of those letter shapes on the keyboard.

gzotti replied:
Yes, I know this is likely slow work with "insert character" from some extended tables. But if made well, it will be a great and very valuable addition. Many thanks!

gzotti commented on skycultures/arabian_peninsula/description.en.utf8:
```diff
@@ -0,0 +1,109 @@
+<h5 id="the-two-little-dogs">The Two Little Dogs</h5>
+<p>An asterism of the two stars: δ CMa and ε CMa. There heliacal rising is used for timing. The name is common in central Arabia.</p>
+<h3 id="star-names">Star Names</h3>
+<h5 id="the-fifth">The Fifth</h5>
```
I think here it would be great to see Arab spelling, English transliteration and English translation.

kajaji replied:
I will implement that if the transliteration standard discussed previously is suitable.

gzotti commented on skycultures/arabic/constellation_names.eng.fab:
```diff
@@ -0,0 +1,138 @@
+1700 "النسر الطائر" _("The Flying Eagle") 1,2,3,4
+1800 "الفوارس" _("The Knights") 1,2,3,4
+1900 "الناقة" _("The She-Camel") 1,2,4
+1901 "رأس الناقة" _("Head Of The She-Came") 1,2,4
```
final letter missing in "Camel" (3x)

kajaji replied:
fixed

gzotti commented on skycultures/arabic/constellation_names.eng.fab:
```diff
@@ -0,0 +1,138 @@
+2702 "كلبا الدبران" _("The Two Dogs Of Al-Dabaran") 1,2,3,4
+2703 "القلاص" _("The Young She-Camels") 1,2,4
+2800 "الأسد" _("The Lion") 1,2,3,4
+2801 "الذراع المقبوضة" _("Retracted Arm") 1,2,3,4
```
also here, add article to be consistent?

kajaji replied:
fixed

kajaji commented:
@gzotti @alex-w 
Thanks for your support.
I have added:

- references to all names of constellations, stars, dso's
- tranliteration to all Arabic names

I added two markdown files to each constellation: a translitration reference and a names dictionary. The names dictionay is a table that has Arabic name, transliteration, English meaning, some extra information, and the list of the references for the name.
I kept this separate from the discription file because of the large size of the table, but Ieft it in the Arabian Penunsila sc because the size is reasonaaable and as a sample.

Tell me what you think.

alex-w commented:
@gzotti please merge this PR if it is ready

gzotti commented:
There may still be occasional obvious typos which I might correct after merge. Apart from that,  this is a major enrichment! The md dictionary files will at least allow us to format the source files when we get a clearer view of what we need or how we want to format it. 

@kajaji many thanks!
